l10n_ec_account_edi/models/account_move.py

```diff
@@ -306,3 +310,14 @@ def _reverse_move_vals(self, default_values, cancel=True):
             l10n_ec_legacy_document_authorization=self.l10n_ec_xml_access_key,
         )
         return move_vals
+
+    def l10n_ec_send_email(self):
+        WizardInvoiceSent = self.env["account.invoice.send"]
+        self.ensure_one()
+        res = self.with_context(discard_logo_check=True).action_invoice_sent()
+        context = res["context"]
+        send_mail = WizardInvoiceSent.with_context(**context).create({})
+        # enviar factura automaticamente por correo
+        # simular onchange y accion
+        send_mail.onchange_template_id()
```

Comment:
Should be better get specific template, with ref of template, users can change templates and sequences

Reply:
I not understand your comment, this function call to `action_invoice_sent` and this function call to function https://github.com/odoo/odoo/blob/15.0/addons/account/models/account_move.py#L3284
`onchange_template_id` is only for render template and fill fields into wizard

this module not create a new email template, but if you need a new template can be done into new module and inherit easily  https://github.com/odoo/odoo/blob/15.0/addons/account/models/account_move.py#L3250-L3258

Reply:
Your are right, there are two templates on account module, it's more like a nice to have